src/main/resources/application.yml

```diff
@@ -1,26 +1,39 @@
 spring:
+  profiles: 
+    active: devqa
   application:
     name: platform.training.bookmiddle
     version: 1.0.0
   jackson:
     default-property-inclusion: non_null
-
+    
 server:
-  port: ${service.port:8080}
+  port: ${service.port:8888}
 
 mongoDB:
   database: bookDB
   host: 127.0.0.1
   port: 27017
   
-sqs:
-  queueUrl: https://sqs.us-east-2.amazonaws.com/404669482207/book-track.fifo
-
 swagger:
   title: BookMiddle Service REST APIs
   description: This successfully showcased Spring Boot REST API
   contact:
     name: Michael
     url: http://www.ebsco.com
     email: mpanson@ebsco.com
-  version: 1.0.0
\ No newline at end of file
+  version: 1.0.0
+---
+spring:
+  profiles: test
+  
+sqs:
+  queueUrl: http://localhost:4576/queue/Megalodon-Queue
+
+---
+spring:
+  profiles: devqa, live, integration
```

Comment:
We should have only two profiles for this app, "test" and "non-test". We are not going to deploy this app to eureka. In other words, when it is not a "test" consider the "default" profile.